tests/math/decomposition_test.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any
+
+import pytest
+
+import jax
+import jax.numpy as jnp
+from jax.experimental import sparse
+
+from ott.math.decomposition import DenseCholeskySolver, SparseCholeskySolver
+
```

Comment:
Please add [this](https://github.com/ott-jax/ott/blob/main/tests/geometry/graph_test.py#L21)